tldraw/packages/react/src/hooks/useSetup.ts

```diff
@@ -54,6 +54,10 @@ export function useSetup<
     if (onPaste) unsubs.push(app.subscribe('paste', onPaste))
     if (onCanvasDBClick) unsubs.push(app.subscribe('canvas-dbclick', onCanvasDBClick))
     // Kind of unusual, is this the right pattern?
-    return () => unsubs.forEach(unsub => unsub())
+    // Allows the app to send the last notice before the unmount
+    return () => setTimeout(
+      () => unsubs.forEach(unsub => unsub()),
+      1000
+    )
```

Comment:
Shall we make the persist on unmount here to make it more clear?

Reply:
Good point!

Reply:
Hmm, I don't think we can make the persist on unmount here because this can be triggered when there's any layout change, right?
I noticed that the app was completely frozen when zooming in/out.